plugins/module_utils/network/eos/argspec/lacp_interfaces/lacp_interfaces.py

```diff
@@ -43,7 +43,11 @@ def __init__(self, **kwargs):
             "options": {
                 "name": {"type": "str"},
                 "port_priority": {"type": "int"},
-                "rate": {"choices": ["fast", "normal"], "type": "str"},
+                "rate": {
+                    "choices": ["fast", "normal"],
+                    "type": "str",
+                    "aliases": ["timer"],
```

Comment:
It looks like "timer" is the key used internally, would it not be more sensible to rename this to "timer" and add the alias "rate" for compatibility?